core/web/eth_keys_controller_test.go

```diff
@@ -3,12 +3,16 @@ package web_test
 import (
```

Comment:
I also considered testing the methods directly (i.e. unit testing `Create`), rather than hitting the API routes(more like integration testing). It's indirect, but possible - I'd have to mock and listen for particular HTTP server responses, and then build asserts around the state of gin. I'm wondering if that signals a deeper smell that my changes didn't properly address - i.e. the APIs should return a standardized response structure, which is then picked up by middleware to turn into the HTTP response.